addons/l10n_dk_oioubl/tests/test_xml_oioubl_dk.py

```diff
@@ -173,6 +176,17 @@ def test_export_partner_fr_without_siret_should_raise_an_error(self):
         with self.assertRaisesRegex(UserError, "The company registry is required for french partner:"):
             self.create_post_and_send_invoice(partner=self.partner_c)
 
+    @freeze_time('2017-01-01')
+    def test_oioubl_export_partner_without_vat_number(self):
+        """ This test verifies that we can't export an OIOUBL file for a partner
+            who doesn't have a tax ID. It verifies that we receive a UserError
+            telling to the user that this field is missing.
+        """
+        self.partner_b.vat = None
+        with self.assertRaises(UserError) as exception:
+            self.create_post_and_send_invoice(partner=self.partner_b)
+        self.assertIn("The field 'Tax ID' is required", exception.exception.args[0])
```

Comment:
Another thing, not sure if it matters (depends on how runbot works). If you install `l10n_latam_base`, the string of the `vat` field is renamed to "Identification Number", which would make this test fail :thinking: .

Reply:
I run the l10n build on runbot to check that. What we can do is getting the string value of the vat field and populate the assert with it :sweat_smile: